plugins/UM3NetworkPrinting/NetworkPrinterOutputDevice.py

```diff
@@ -1114,6 +1115,11 @@ def _onFinished(self, reply):
                     else:
                         Logger.log("w", "Unable to save authentication for id %s and key %s", self._authentication_id, self._getSafeAuthKey())
 
+                    # Request 'system' printer data once, when we know we have authentication, so we know we can set the system time.
```

Comment:
This file makes me sad, but I guess adding it here doesn't make it any worse, 
I assume the auth/verify is always called when opening a printer/starting up?

Reply:
Yep, its always called. This file also makes me sad :(